README.md

```diff
@@ -505,6 +506,15 @@ LevelDB is thread-safe but is **not** suitable for accessing with multiple proce
 
 See the <a href="https://github.com/level/levelup/wiki/Modules"><b>wiki</b></a> for some LevelUP extensions, including [multilevel](https://github.com/juliangruber/multilevel), that may help if you require a single store to be shared across processes.
 
+<a name="typings"></a>
+Typescript 
+----------
+
+LevelUP comes with typescript definitions that can automatically infer options based on a typed `abstract-leveldown` implementation. 
```

Comment:
Maybe the simpler form: "infer options from a typed"

Reply:
done